### DISCUSSION TITLE
Why is code from imported file not executed in server API route ?

### DISCUSSION THREAD

**Discussion by lewebsimple** [hayes/pothos#361] [Q&A] [answered]
I'm trying to define a modular GraphQL schema as suggested [here](https://github.com/hayes/pothos/discussions/273#discussioncomment-2046397).

Let's say we have `server/api/hello.ts` like so:
```
import '../test.ts`;
console.log('Hello from server/api/hello.ts');
export default () => 'Hello World';
```

...and `server/test.ts` only contains `console.log('Hello from server/test.ts');`

Why is the console only outputting "Hello from server/api/hello.ts" (twice?) and not executing the other `console.log` ?

**Comment by hayes** [comment, marked as answer]
This doesn't sound specific to Pothos.  It's possible that what ever server you are using is doing treeshaking on the imported files and not including the console.logs

**Reply by lewebsimple** [reply]
Oops posted in the wrong repository!
